endpoints/api/mirror.py

```diff
@@ -291,6 +299,12 @@ def post(self, namespace_name, repository_name):
 
         data = request.get_json()
 
+        if data["skopeo_timeout_interval"] < 300:
```

Comment:
`skopeo_timeout_interval` is not marked as required, if the parameter is sent with the request will this fail? should we check the existence of `skopeo_timeout_interval` first?

Reply:
@bcaton85 I don't think so because of the `validate_json_request` wrapper that is being called. If the parameter is missing, it should cancel the request. I'll double check to be sure.

https://github.com/quay/quay/blob/140ee4ab800f379d5cfe9f06c3266600ed092386/endpoints/api/__init__.py#L589

Reply:
@bcaton85 yeah, I forgot to add the property to the list of required properties, you were correct. Now it should work as intended.